modules/tinymce/tools/docs/tinymce.js

```diff
@@ -4,14 +4,15 @@
  * @static
  * @class tinymce
  * @borrow-members tinymce.EditorManager
+ * @borrow-members tinymce.EditorObservable
```

Comment:
I missed this earlier, but this is incorrect. `EditorObserable` is only used for the `Editor` not the global tinymce variable.
```suggestion
```